net/znc/Makefile

```diff
@@ -277,7 +278,7 @@ CONFIGURE_VARS += \
 	LIBS="-lstdc++ -lm -lssl -lcrypto $(LIBGCC_S) -lc"
 
 CONFIGURE_ARGS += \
-	--disable-charset \
+	$(if $(CONFIG_ZNC_ICU), --enable-charset, --disable-charset))
```

Comment:
Yes, people do! :)

I'm just an amateur and a newbie, but I do learn by mine. ;)